docs/concepts/login.md

```diff
@@ -1,2 +1,74 @@
 (login)=
-# {octicon}`sign-in` Login
\ No newline at end of file
+# {octicon}`sign-in` Login
+
+Finetuner, as part of the [Jina AI](https://jina.ai/) stack,
+uses [Jina AI Cloud](https://cloud.jina.ai/) for authentication.
+
+Accessing Jina AI Cloud is straightforward and you can do so by following these steps:
+
+```python
+import finetuner
+
+finetuner.login()
+```
+
+Once you execute the `~finetuner.login()` method,
+a browser window will appear, offering various login options.
+After completing the login process, you will receive the following message in your terminal:
+
+```bash
+Your browser is going to open the login page.
+If this fails please open the following link: ...
+🔐 Successfully logged in to Jina AI as [YOUR-USERNAME] (username: your-username)!
+```
+
+```{admonition} What happens when I login?
+:class: hint
+Finetuner will generate a temporary `auth_token` that is associated with your personal account.
+This `auth_token` will be utilized by Finetuner to carry out various tasks such as:
+
+1. Sending your training data to [Jina AI Cloud Storage](https://cloud.jina.ai/user/storage).
+2. Setting up GPU/CPU resources for performing fine-tuning tasks.
+3. Storing fine-tuned models (referred to as Artifacts) on the Jina AI Cloud.
+4. Performing billing (applicable only when the user exhausts their free resources).
+
+By default, the data and fine-tuned models are only visible to you, unless you choose to publish them manually by setting the `public` flag.
+```
+
+## Namespace
+
+Users of Jina AI Cloud have a namespace that is linked to their personal account.
+This namespace ensures that even if data or artifacts share the same name,
+they are separated. 
+
+If you want to download public data or models published by other users,
+you must add `[namespace]/` before the name of the data or artifact.
+For example, `finetuner/quora-train-da`,
+where `finetuner` is the namespace for the account, and `quora-train-da` is the name of the dataset,
+which refers to the quora training set.
```

Comment:
Are all the public resources under the "finetuner" namespace? If so, there is a much easier way to frame this.

Reply:
yes this is a public resource @scott-martens